code/datums/wounds/_wounds.dm

```diff
@@ -0,0 +1,312 @@
+	var/processes = FALSE
+	/// We stop processing on dismembered or detached limbs, but if some other dumb sucker gets a rotten arm attached to them, we use this to remember if we were processing before
+	var/hibernate
+
+	/// How much we're contributing to this limb's bleed_rate
+	var/blood_flow
+
+	/// The list of wounds it belongs in, WOUND_TYPE_BONE, WOUND_TYPE_CUT, or WOUND_TYPE_BURN
+	var/wound_type
+
+	/// The minimum we need to roll on [/obj/item/bodypart/proc/check_wounding()] to begin suffering this wound, see check_wounding_mods() for more
+	var/threshold_minimum
+	/// How much having this wound will add to all future check_wounding() rolls on this limb, to allow progression to worse injuries with repeated damage
+	var/threshold_penalty
+
+	/// What status effect we assign on application
+	var/status_effect_type
+	/// If TRUE and an item that can treat multiple different types of coexisting wounds (gauze can be used to splint broken bones, staunch bleeding, and cover burns), we get first dibs if we come up first for it, then become nonpriority.
+	/// Otherwise, if no untreated wound claims the item, we cycle through the non priority wounds and pick a random one who can use that item.
+	var/treat_priority = FALSE
+
+	/// If having this wound makes currently makes the parent bodypart unusable
+	var/disabling
+
+	var/datum/status_effect/linked_status_effect
+
+	/// if you're a lazy git and just throw them in cryo, the wound will go away after accumulating severity * 25 power
+	var/cryo_progress
+
+	var/list/scarring_descriptions = list("general disfigurement")
+	/// If we've already tried scarring while removing (since remove_wound calls qdel, and qdel calls remove wound, .....) TODO: make this cleaner
+	var/already_scarred = FALSE
+
+/datum/wound/Destroy()
+	if(limb && limb.wounds && (src in limb.wounds)) // destroy can call remove_wound() and remove_wound() calls qdel, so we check to make sure there's anything to remove first
+		remove_wound()
+	limb = null
+	victim = null
+	return ..()
+
+/**
+  * apply_wound() is used once a wound type is instantiated to assign it to a bodypart, and actually come into play.
+  *
+  *
+  * Arguments:
+  * * L: The bodypart we're wounding, we don't care about the person, we can get them through the limb
+  * * silent: Not actually necessary I don't think, was originally used for demoting wounds so they wouldn't make new messages, but I believe old_wound took over that, I may remove this shortly
+  * * old_wound: If our new wound is a replacement for one of the same time (promotion or demotion), we can reference the old one just before it's removed to copy over necessary vars
+  */
+/datum/wound/proc/apply_wound(obj/item/bodypart/L, silent = FALSE, datum/wound/old_wound = null)
+	if(!istype(L) || !L.owner || !(L.body_zone in viable_zones))
+		qdel(src)
+		return
+
+	if(ishuman(L.owner))
+		var/mob/living/carbon/human/H = L.owner
+		if(organic_only && (NOBLOOD in H.dna.species.species_traits))
```

Comment:
you could check inherent_biotypes for MOB_ORGANIC instead of species_traits for NOBLOOD

Reply:
alternatively, you can check the type of the LIMB (instead of the mob) is BODYPART_ORGANIC (vs. BODYPART_ROBOTIC)